src/components/icons/ArrowRightIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import { forwardRef, Icon, IconProps } from '@chakra-ui/react';
+import { FunctionComponent } from 'react';
+
+const ArrowRightIcon: FunctionComponent<IconProps> = forwardRef((props, ref) => (
+  <Icon viewBox="0 0 20 25" ref={ref} {...props}>
+    <path
+      d="M2.57227 12.4122H17.4297M17.4297 12.4122L10.9069 18.9349M17.4297 12.4122L10.9069 5.8894"
+      // @todo stroke="currentColor" doesn't work
```

Comment:
`fill="currentColor"`

Reply:
doesn't work when using fill & color (as suggested in the other conversation) but works with stroke & color so I leave it with the working solution :)